esmlab/core.py

```diff
@@ -504,9 +493,9 @@ def compute_mon_anomaly(self, slice_mon_clim_time=None):
         # reset month to become a variable
         computed_dset = computed_dset.reset_coords('month')
 
-        computed_dset[self.time_coord_name].data = self.time.data
+        computed_dset[self.time_coord_name] = self.time
```

Comment:
keep `.data` on RHS? But it seems like you're rewriting attributes later, in which case this might not matter.